alpine-rails-nginx/install-nginx.sh

```diff
@@ -17,7 +17,7 @@ function verified_curl {
 }
 
 mkdir /usr/local/src
-cd /usr/local/src
+cd /usr/local/src || exit 1
```

Comment:
I see that this file includes `strict-mode`, why do you need to fail explicitly?